ocs_ci/helpers/managed_services.py

```diff
@@ -22,6 +22,74 @@
 log = logging.getLogger(__name__)
 
 
+def get_machine_pool_info_list():
+    """
+    Get the machine pool info list
+
+    Returns:
+        list: The machine pool info list
+
+    """
+    cmd = f"rosa list machinepool --cluster={config.ENV_DATA['cluster_name']} -o yaml"
+    out = run_cmd(cmd)
+    return yaml.safe_load(out)
+
+
+def get_ceph_osd_nodepool_info():
```

Comment:
Can this and following functions be moved to ocs_ci/ocs/rosa.py?

Reply:
Here you meant the file `ocs_ci/utility/rosa.py`, right?